test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -675,6 +677,32 @@ UCS_TEST_P(test_ucp_wireup_2sided, no_loopback_with_delay) {
     test_connect_loopback(true, false);
 }
 
+UCS_TEST_P(test_ucp_wireup_2sided, async_connect) {
+    if (!(GetParam().ctx_params.features & UCP_FEATURE_TAG)) {
+        UCS_TEST_SKIP_R("The test requires UCP_FEATURE_TAG");
+    }
+    sender().connect(&receiver(), get_ep_params());
+    ucp_ep_h send_ep = sender().ep();
+    std::vector<void *> reqs;
+
+    reqs.push_back(ucp_tag_send_nb(send_ep, NULL, 0, DT_U64, 1, send_completion));
+    EXPECT_FALSE(UCS_PTR_IS_ERR(reqs.back()));
+
+
+    ucs_time_t deadline = ucs_get_time() +
+                          ucs_time_from_sec(UCP_TEST_TIMEOUT_IN_SEC);
```

Comment:
1. ~need to use ucs_time_multiplier()~
2. can you pls add helper function ucs::get_deadline(timeout=UCP_TEST_TIMEOUT_IN_SEC) - to calc deadline time, with ucs_time_multiplier()
3. can you pls increase UCP_TEST_TIMEOUT_IN_SEC to 60.0